.travis.yml

```diff
@@ -22,23 +22,21 @@ before_install:
 
 install:
   - until sudo apt-get -qq update; do echo retry; done
-  - until sudo apt-get -qq install libstdc++5; do echo retry; done
-  - until sudo apt-get -qq install g++-5; do echo retry; done
+  - until sudo apt-get -qq install libstdc++5 g++-5 ninja-build libboost-all-dev; do echo retry; done
   ### precompiled binaries for LLVM 3.8  
   - export LLVM_ROOT="$TRAVIS_BUILD_DIR/LLVM-3.8-Linux"
   - wget https://jorgenavas.github.io/sw/LLVM-3.8-Linux.tar.gz  
   - tar xf LLVM-3.8-Linux.tar.gz
-  ### BOOST
-  - until sudo apt-get -qq install libboost-all-dev; do echo retry; done
 
 script:
   - mkdir -p build
   - cd build
-  - /usr/bin/cmake -DBOOST_ROOT=${BOOST_ROOT} -DLLVM_DIR=${LLVM_ROOT}/share/llvm/cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=g++-5 -DCMAKE_PROGRAM_PATH=/usr/bin  -DCMAKE_INSTALL_PREFIX=run ../;
+  - /usr/bin/cmake -GNinja -DBOOST_ROOT=${BOOST_ROOT} -DLLVM_DIR=${LLVM_ROOT}/share/llvm/cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_COMPILER=g++-5 -DCMAKE_PROGRAM_PATH=/usr/bin  -DCMAKE_INSTALL_PREFIX=run ../;
```

Comment:
Does it make a difference in travis? I thought that Travis runs everything with one core anyway.

Reply:
I'm not sure, but we use ninja with seahorn.

Reply:
To keep minimal the `.travis.yml` I think we don't need ninja since it won't have an impact.
btw, I use ninja but some people do not like so we shouldn't not force people to do that.